bookwyrm/templates/guided_tour/lists.html

```diff
@@ -0,0 +1,214 @@
+                    </code>",
+            title: "{% trans 'Creating a new list' %}",
+            attachTo: {
+                element: "#tour-create-list",
+                on: "left",
+            },
+            buttons: [
+                {
+                    action() {
+                        return this.back();
+                    },
+                    secondary: true,
+                    text: "{% trans 'Back' %}",
+                },
+                {
+                    action() {
+                        return this.next();
+                    },
+                    text: "{% trans 'Next' %}",
+                },
+            ],
+        },
+        {
+            text: "{% trans 'Give your list a name and can optionally give it a description to help other people understand what your list is about.' %}",
+            title: "{% trans 'Creating a new list' %}",
+            attachTo: {
+                element: "#tour-list-name",
+                on: "top",
+            },
+            buttons: [
+                {
+                    action() {
+                        return this.back();
+                    },
+                    secondary: true,
+                    text: "{% trans 'Back' %}",
+                },
+                {
+                    action() {
+                        return this.next();
+                    },
+                    text: "{% trans 'Next' %}",
+                },
+            ],
+        },
+        {
+            text: "{% trans 'Choose who can see your list here. List privacy can be ' %}\
+            <strong>\
+                {% trans 'Public' %}\
+            </strong>\
+            {% trans ' (everyone can see), ' %}\
+            <strong>\
+                {% trans 'Unlisted' %}\
+            </strong>\
+            {% trans ' (everyone can see, but it doesn\'t appear on the list discovery page), ' %}\
+            <strong>\
+                {% trans 'Followers' %}\
+            </strong>\
+            {% trans ' (only your followers can see), or ' %}\
+            <strong>\
+                {% trans 'Private' %}\
+            </strong>\
+            {% trans '  (only you can see)' %}",
+            title: "{% trans 'List privacy' %}",
+            attachTo: {
+                element: "#tour-privacy-select",
+                on: "left",
+            },
+            buttons: [
+                {
+                    action() {
+                        return this.back();
+                    },
+                    secondary: true,
+                    text: "{% trans 'Back' %}",
+                },
+                {
+                    action() {
+                        return this.next();
+                    },
+                    text: "{% trans 'Next' %}",
+                },
+            ],
+        },
+        {
+            text: "{% trans 'You can also decide how your list is to be curated - only by you, by anyone, or by a group.' %}\
```

Comment:
I think this part of the tour would be easier to follow if it steps you through a simpler list create process, rather than nesting learning about groups in learning about creating lists. Maybe instead it could do groups first, and then lists, so the group is already available?

Reply:
Originally I did it that way! But I felt that learning shelves and adding a book is likely the first thing people want to know, and lists naturally follows from that, so doing groups first seemed a bit obtuse. Will think a bit more about this, the right order is tricky.